### PR TITLE
Add of a page "unknown technology" linked to technlogy detection error

### PR DESCRIPTION
**Pull request by Zyko0**

### CODE REVIEW

File: _posts/deployment/2015-11-02-unknown-detection.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Solutions
+
+### Project in a subdirectory of the git repository
+
+If the project you want to deploy is not at the root of your GIT repository, you need to define the `PROJECT_DIR` the environment variable ([see documentation]({% post_url /deployment/2015-06-02-project-dir %})).
+
+### Technology detection
+
+See how we detect your technology:
```
Comment (yannski) [inline _posts/deployment/2015-11-02-unknown-detection.md]: please list explictly the different ways and refer to other pages just context information

File: _posts/deployment/2015-11-02-unknown-detection.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+* [Clojure]({% post_url /languages/clojure/2015-01-26-clojure %})
+* [Erlang]({% post_url /languages/erlang/2015-01-26-erlang %})
+* [Go]({% post_url /languages/go/2014-12-22-go %})
+* [Groovy]({% post_url /languages/groovy/2015-01-26-groovy %})
+* [Haskell]({% post_url /languages/haskell/2015-01-26-haskell %})
+* [Java]({% post_url /languages/java/2015-09-01-java %})
+* [Java]({% post_url /languages/java/2015-09-01-java %})
```
Comment (Soulou) [inline _posts/deployment/2015-11-02-unknown-detection.md]: Double

File: _posts/deployment/2015-11-02-unknown-detection.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: Unknown Technology
```
Comment (Soulou) [inline _posts/deployment/2015-11-02-unknown-detection.md]: File name doesn't match title, correct file name should be unknown-technology

File: _posts/deployment/2015-11-02-unknown-detection.md (hunk 1 above)
Comment (Soulou) [inline _posts/deployment/2015-11-02-unknown-detection.md]: Maybe sort the languages from the more common to the least

Javascript/node/meteor, PHP, Ruby, Python, Java, Go, then the others.

**Comment by Zyko0** [comment]
Can we merge this?

**Pull request merged**